java/ci/Dockerfile.rocky

```diff
@@ -26,7 +26,7 @@ ARG TARGETPLATFORM=linux/amd64
 # multi-platform build with: docker buildx build --platform linux/arm64,linux/amd64 <ARGS> on either amd64 or arm64 host
 # check available official arm-based docker images at https://hub.docker.com/r/nvidia/cuda/tags (OS/ARCH)
 FROM --platform=$TARGETPLATFORM nvidia/cuda:$CUDA_VERSION-devel-rockylinux$OS_RELEASE
-ARG TOOLSET_VERSION=11
+ARG TOOLSET_VERSION=14
 ### Install basic requirements
 RUN dnf --enablerepo=powertools install -y  scl-utils gcc-toolset-${TOOLSET_VERSION} git zlib-devel maven tar wget patch ninja-build boost-devel
```

Comment:
Please add `gcc-toolset-11` to ensure compatibility here (let’s aim to always keep two toolsets installed), thanks~

Reply:
Done.